src/NonTransactionalSagas.StorageTable.AcceptanceTests/ConfigureAzureStoragePersistence.cs

```diff
@@ -13,11 +13,6 @@ Task IConfigureEndpointTestExecution.Configure(string endpointName, EndpointConf
             var sagaPersistence = configuration.UsePersistence<AzureTablePersistence, StorageType.Sagas>()
                 .UseTableServiceClient(SetupFixture.TableServiceClient);
 
-            if (!(settings.TryGet("allowTableCreation", out bool allowTableCreation) && allowTableCreation))
-            {
-                sagaPersistence.DisableTableCreation();
-            }
-
```

Comment:
Was this supposed to be removed entirely, or converted to using the new type?

Reply:
Removed as it wasn't used in any of the tests